.github/workflows/linux-test.yml

```diff
@@ -66,6 +70,7 @@ jobs:
       REF: ${{ inputs.ref }}
       CU_VERSION: ${{ matrix.desired_cuda }}
       SCRIPT: ${{ inputs.script }}
+      CHANNEL: ${{ inputs.channel }}
```

Comment:
Does this do anything?

Reply:
Initially, I thought it would affect the `PIP_INSTALL_TORCH` command generated, but I will try omitting it and see how the CI performs.